types/targets.pp

```diff
@@ -0,0 +1,14 @@
+# @summary a list of available targets per chain and table
+type Ferm::Targets::Raw::PRERROUTING    = Enum['RETURN', 'ACCEPT', 'DROP', 'LOG', 'NOTRACK']
```

Comment:
I'm not sure if the styleguide suggests one file per type, that might be the case.